app/src/components/ServerSettings/ModelManagement.tsx

```
@@ -1100,3 +1052,106 @@ export function ModelManagement() {
+      ? 'Downloading'
+      : model.downloaded
+        ? 'Downloaded'
+        : 'Not downloaded';
+  const sizeText =
+    model.downloaded && model.size_mb && !showDownloading ? `, ${formatSize(model.size_mb)}` : '';
+  const rowLabel = `${model.display_name}, ${statusText}${sizeText}. Use Tab to reach Download or Delete.`;
+
+  return (
+    <div
+      className="flex items-center justify-between p-3 border rounded-lg"
+      role="group"
```

Biome (2.4.14)

[error] The elements with this role can be changed to the following elements:
<fieldset>

(lint/a11y/useSemanticElements)